universal-login-relayer/src/integration/ethereum/WalletDeployer.ts

```diff
@@ -1,22 +1,16 @@
 import {Contract, Wallet} from 'ethers';
 import {TransactionOverrides} from '@universal-login/commons';
-import {beta2} from '@universal-login/contracts';
-
-interface DeployFactoryArgs {
-  publicKey: string;
-  intializeData: string;
-  signature: string;
-}
+import {ProxyFactoryInterface} from '@universal-login/contracts';
```

Comment:
should it be in gnosis.interfaces?

Reply:
It might be, but I don't think that doing it in this big PR is the best idea. I would prefer to move that kind of improvements to post-gnosis migration refactoring.

Reply:
@xitronix good point, let's do it in follow-up PR